### DISCUSSION TITLE
Issue with opening connection

### DISCUSSION THREAD

**Discussion by littlebear1123** [iterate-ch/cyberduck#15718] [Q&A]
When I opened a connectio, there was a yellow exclamation ⚠️ sign at the right of the URL address. 

Is anyone here cha help me? I just downloaded the Duck today.

**Comment by tatze96** [comment]
The ⚠️ sign next to the URL indicates that the current URL may not be accessible. This can be the case if there is no hostname set within the *Server* field or the set combination of hostname, credentials, and path leads to an invalid URL.

To connect to the server, check that you've selected the correct protocol and added the needed server information (hostname, credentials, path, etc.) into the corresponding fields. Contact your server administrator for further information if you are unsure about the needed hostname or the corresponding credentials.
